graphql/schema.graphql

```diff
@@ -277,6 +305,16 @@ type Mutation {
   "Updates a user record."
   updateUser(user: UserInputType!): User!
 
+  "Upserts a user personalization record."
+  upsertUserPersonalization(
+    userPersonalization: UserPersonalizationInputType!
+  ): UserPersonalization!
+
+  "Updates a user personalization record."
+  updateUserPersonalization(
+    userPersonalization: UserPersonalizationInputType!
+  ): UserPersonalization!
```

Comment:
Is `update` needed if we already have `upsert`?

Reply:
Upsert was added for user_personalization because User had both upsert and update.
Notably, in this case, given that the upsert calls update, update will be needed.
Hope this provides context as to why update is still needed.
Thanks.